src/igvfd/types/file_set.py

```diff
@@ -37,14 +37,18 @@ class FileSet(Item):
         'control_for': ('FileSet', 'control_file_sets')
     }
     embedded_with_frame = [
-        Path('award', include=['@id', 'component']),
+        Path('award', include=['@id', 'component', 'contact_pi']),
+        Path('award.contact_pi', include=['@id', 'title']),
         Path('lab', include=['@id', 'title']),
         Path('submitted_by', include=['@id', 'title']),
         Path('files', include=['@id', 'accession', 'aliases']),
         Path('control_for', include=['@id', 'accession', 'aliases']),
         Path('donors', include=['@id', 'accession', 'aliases', 'taxa']),
-        Path('samples', include=['@id', 'accession', 'aliases',
-             'sample_terms', 'summary', 'taxa']),]
+        Path('samples', include=['@id', 'accession', 'aliases', 'treatments', 'cell_fate_change_treatments',
+             'classification', 'disease_terms', 'modifications', 'sample_terms', 'summary', 'targeted_sample_term', 'taxa']),
+        Path('samples.sample_terms', include=['@id', 'term_name']),
+        Path('samples.targeted_sample_term', include=['@id', 'term_name']),
+    ]
 
```

Comment:
I think you can have only 2 entries instead of 3.
Path('samples.sample_terms', include=['@id', 'term_name', 
'classification', 'disease_terms', 'modifications', '', 'summary', '', 'taxa' , 'accession', 'aliases', 'treatments', 'cell_fate_change_treatments',
]),
      Path('samples.targeted_sample_term', include=['@id', 'term_name']),